1-js/02-first-steps/11-logical-operators/article.md

```diff
@@ -153,137 +153,135 @@ alert( true && false );  // false
 alert( false && false ); // false
 ```
 
-An example with `if`:
+Օրինակ `if`֊ի հետ։
 
 ```js run
-let hour = 12;
-let minute = 30;
+let hour = 12
+let minute = 30
 
 if (hour == 12 && minute == 30) {
-  alert( 'The time is 12:30' );
+  alert('Ժամը 12։30֊ն է')
 }
 ```
 
-Just as with OR, any value is allowed as an operand of AND:
+Ինչպես ԿԱՄ֊ի դեպքում, ԵՎ֊ի օպերանդները նույնպես կարող են ունենալ կամայական արժեք․
 
 ```js run
-if (1 && 0) { // evaluated as true && false
-  alert( "won't work, because the result is falsy" );
+if (1 && 0) { // հաշվվում է ինչպես true && false
+  alert('չի երևա, որովհետև արժեքը սխալական է')
 }
 ```
 
+## ԵՎ "&&"֊ը գտնում է առաջին սխալական արժեքը
 
-## AND "&&" finds the first falsy value
-
-Given multiple AND'ed values:
+Տրված է մի քանի արժեքներ կապակցված ԵՎ֊ով․
 
 ```js
-result = value1 && value2 && value3;
+result = value1 && value2 && value3
 ```
 
-The AND `&&` operator does the following:
+ԵՎ `&&` օպերատորը կատարում է հետևյալ գործողություննեը․
 
-- Evaluates operands from left to right.
-- For each operand, converts it to a boolean. If the result is `false`, stops and returns the original value of that operand.
-- If all operands have been evaluated (i.e. all were truthy), returns the last operand.
+- Հաշվում է օպերանդների արժեքները ձախից աջ։
+- Ամեն օպերանդի արժեք փոխակերպում է տրամաբանական տիպի։ Եթե արդյունքը `false` է այն կանգ է առնում և վերադարձնում այդ օպերանդի օրիգինալ արժեքը։
+- Եթե բոլոր օպերանդները հաշվվում են, և բոլորի արժեքը `true` է լինում, ապա վերադարձնում է վերջին օպերանդի արժեքը։
 
-In other words, AND returns the first falsy value or the last value if none were found.
+Այլ կերպ ասած, ԵՎ֊ը վերադարձնում է առաջին սխալական արժեքը, կամ վերջին արժեքը, եթե ոչ մի սխալական արժեք չի գտնվել։
 
-The rules above are similar to OR. The difference is that AND returns the first *falsy* value while OR returns the first *truthy* one.
+Վերոնշյալ կանոնները նման են ԿԱՄ֊ին։ Տարբերությունն այն է, որ ԵՎ֊ը վերադարձնում է առաջին _սխալական_ արժեքը, իսկ ԿԱՄ֊ը՝ առաջին _ճշմարիտ_ արժեքը։
 
-Examples:
+Օրինակներ․
 
 ```js run
-// if the first operand is truthy,
-// AND returns the second operand:
-alert( 1 && 0 ); // 0
-alert( 1 && 5 ); // 5
-
-// if the first operand is falsy,
-// AND returns it. The second operand is ignored
-alert( null && 5 ); // null
-alert( 0 && "no matter what" ); // 0
+// Եթե առաջին օպերանդը ճշմարիտ է,
+// ապա ԵՎ֊ը վերադարձնում է երկրոդ օպերանդը։
+alert(1 && 0) // 0
+alert(1 && 5) // 5
+
+// Եթե առաջին օպերանդը սխալական է,
+// ապա ԵՎ֊ը վերադարձնում է այն։ Երկրորդ օպերանդը անտեսվում է։
+alert(null && 5) // null
+alert(0 && 'կապ չունի ինչ') // 0
```

Comment:
```suggestion
alert( null && 5 ); // null
alert( 0 && "կապ չունի ինչ" ); // 0
```
Please make your it is not your editor altering those examples. Also make sure to restore the original formatting of any example code in all files of this PR. Thank you.